### ISSUE TITLE
[AUTOMATION]: Update Nightwatch axeCheckJS to capture failing tests

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#21738]
# [508-defect-1 :exclamation: Launchblocker](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Details
While auditing https://github.com/department-of-veterans-affairs/va.gov-team/issues/21073 with the VSA specialists, we noticed an axe-core violation that wasn't appearing in our Nightwatch axeChecks. Further investigation revealed the axeCheckJS helper was failing axe tests silently.

## Acceptance Criteria

- [ ] Nightwatch axeCheckJS helper fails tests correctly
- [ ] All breaking tests are verified as pages that are in production
- [ ] Failing page tests are corrected
- [ ] Nightwatch does not throw false positive axe violations

## Screenshots or Trace Logs

![Screen Shot 2021-03-18 at 12 49 46 PM](https://user-images.githubusercontent.com/934879/111673108-a24c3d80-87e8-11eb-9d6a-b143c140670b.png)

---
<img width="862" alt="Screen Shot 2021-03-18 at 12 50 03 PM" src="https://user-images.githubusercontent.com/934879/111673153-ad9f6900-87e8-11eb-8f38-bf8d373acbab.png">

**Comment by 1Copenut** [comment]
Closed out with two PRs:

* https://github.com/department-of-veterans-affairs/vets-website/pull/16424
* https://github.com/department-of-veterans-affairs/content-build/pull/151
